adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala

```diff
@@ -3051,4 +3242,44 @@ class ADAMContext(@transient val sc: SparkContext) extends Serializable with Log
       loadParquetFragments(pathName, optPredicate = optPredicate, optProjection = optProjection)
     }
   }
+
+  /**
+   * Return integer size of partitions if the specified path of Parquet + Avro files is partitioned.
```

Comment:
Suggest clarifying from:

> Return integer size of partitions if the specified path of Parquet + Avro files is partitioned.

to:

> Return length of partitions in base pairs if file is partitioned.

Otherwise, it is unclear as to wether this returns the number of partitions or the size of a single partition (and in the latter case, what the "size" means).

Reply:
Done